Makefile

```diff
@@ -63,9 +64,11 @@ else
 	yarn dist
 endif
 
+.PHONY: $(extension_node_modules)
 $(extension_node_modules):
 	cd $(@:/node_modules=) && npm install --no-audit --no-fund
 
+.PHONY: $(extension_dists)
```

Comment:
This makes no sense to me, phony will trigger `npm run build` always -> multiple times in CI. See #1407 .

Reply:
Maybe I was wrong about that one. I know that the other two `.PHONY` rules are required but I am now less sure about this one. 

How about I remove this line and keep the other two?

Reply:
`$(extension_node_modules):` is also super slow (because of npm).. IMHO phony is not good fit there. Top level phony is probably ok because `yarn install` is fast (not sure about `yarn check`).

Reply:
Then we should use `yarn install` (and then `.gitignore` the yarn lock file). `.PHONY` is exactly what this is for because `make`'s rule about when to rebuild a dependency is entirely based on filesystem information.